NCS.DSS.AzureSearchUtility/Helpers/IndexModelHelper.cs

```diff
@@ -14,26 +15,38 @@ public static Index CreateIndexModelForCustomer(string indexName)
                 Name = indexName,
                 Fields = new[]
                 {
```

Comment:
Consider refactoring this code block into a function to improve code maintenance and readability.